### PR TITLE
Delete forbidden pairs

### PR DESCRIPTION
**Pull request by nemanja777**
post request delete

### CODE REVIEW

**Comment by Zaista** [comment]
Name and description of the request are still the same. Change it to something meaningful please

File: public/js/santaAdmin.js [hunk 1]
```diff
@@ -73,8 +73,19 @@ $(async function() {
   $.getJSON(`api/forbidden?groupId=${groupId}`, function(result) {
     // TODO make this beautiful
     result.forEach((pair, index) => {
-      $('#forbiddenPairsTable tbody').append(`<tr><td><b>${++index}</b></td><td>${pair.forbiddenPair1}</td><td>${pair.forbiddenPair2}</td></tr>`);
+      $('#forbiddenPairsTable tbody').append(`<tr><td><b>${++index}</b></td><td class="pairColumn1">${pair.forbiddenPair1}</td><td>${pair.forbiddenPair2}</td><td><i class="buttonDelete bi bi-trash" style='color:red'></i></td></tr>`);
     });
+    $('.buttonDelete').on('click', function ()  {
+        const pair = {
```
Comment (Zaista) [inline public/js/santaAdmin.js:79]: What's this used for?

File: public/js/santaAdmin.js [hunk 2]
```diff
@@ -73,8 +73,19 @@ $(async function() {
...
+              forbiddenUser1: $('#forbiddenUser1').val(),
+              forbiddenUser2: $('#forbiddenUser2').val()
+            };
+            $.post(`api/remove`, { forbiddenPairId: $(this).parents('.pairColumn1').html() }, result => {
+                  showAlert(true, result.success);
```
Comment (Zaista) [inline public/js/santaAdmin.js:84]: what's this used for?

File: public/js/santaAdmin.js [hunk 3]
```diff
@@ -73,8 +73,19 @@ $(async function() {
...
+                  // TODO reload page or add item to the table manually, close modal
```
Comment (Zaista) [inline public/js/santaAdmin.js:85]: What do you mean by this?

File: public/js/santaAdmin.js [hunk 4]
```diff
@@ -73,8 +73,19 @@ $(async function() {
...
+                });
+                return false;
```
Comment (Zaista) [inline public/js/santaAdmin.js:87]: What's this used for?

File: public/js/santaAdmin.js [hunk 5]
```diff
@@ -96,6 +107,7 @@ $(async function() {
     return false;
   });
 
+
```
Comment (Zaista) [inline public/js/santaAdmin.js:110]: remove this

File: public/santaAdmin.html [hunk 6]
```diff
@@ -92,13 +93,17 @@ <h4>User access</h4>
 
                 <div class="mt-5" id="item-user-settings">
                     <h4>User settings</h4>
-                    <p>Settings related to the SecretSanta users. Deactivated users will stop being part of the SecretSanta group,
-                        they will not be drafted in the next year, and they will stop receiving any emails. 
-                        They will however show up in the history. If you wish to change this, please remove them manually
+                    <p>Settings related to the SecretSanta users. Deactivated users will stop being part of the
+                        SecretSanta group,
```
Comment (Zaista) [inline public/santaAdmin.html:97]: ???

**Review by Zaista — CHANGES_REQUESTED** [review]
Fix the highlighted changes

File: public/js/santaAdmin.js (hunk 1 above)
Comment (nemanja777) [reply public/js/santaAdmin.js:79]: We use the const keyword in JavaScript to declare variables whose value can be initialized only at the time of declaration.

File: public/js/santaAdmin.js (hunk 2 above)
Comment (nemanja777) [reply public/js/santaAdmin.js:84]: We use to display an alert window that contains a specified message

File: public/js/santaAdmin.js (hunk 1 above)
Comment (Zaista) [reply public/js/santaAdmin.js:79]: What I meant was: what's this object `pair` used for? Why did you create it? What's its purpose?

File: public/js/santaAdmin.js (hunk 2 above)
Comment (Zaista) [reply public/js/santaAdmin.js:84]: Did we agree on that? That was not part of the task as far as I recall.

File: public/js/santaAdmin.js [hunk 7]
```diff
@@ -73,9 +73,15 @@ $(async function() {
   $.getJSON(`api/forbidden?groupId=${groupId}`, function(result) {
     // TODO make this beautiful
     result.forEach((pair, index) => {
-      $('#forbiddenPairsTable tbody').append(`<tr><td><b>${++index}</b></td><td>${pair.forbiddenPair1}</td><td>${pair.forbiddenPair2}</td></tr>`);
+      $('#forbiddenPairsTable tbody').append(`<tr id="forbiddenPairId"><td><b>${++index}</b></td><td class="pairColumn1">${pair.forbiddenPair1}</td><td>${pair.forbiddenPair2}</td><td class="letsTry"><i class="buttonDelete bi bi-trash" style="cursor:pointer; color:red"></i></td><td>${pair._id}</td></tr>`);
```
Comment (Zaista) [inline public/js/santaAdmin.js:76]: this id here in `<tr id="forbiddenPairId">`, I don't see it used anywhere, remove it if not needed

**Review by Zaista — CHANGES_REQUESTED** [review]
Couple of small issues remaining

**Review by Zaista — APPROVED** [review]

**Pull request merged**